src/settings/LoanPolicy/components/EditSections/RequestManagementSection/RequestManagementSection.test.js

```diff
@@ -0,0 +1,456 @@
+          beforeEach(() => {
+            render(
+              <RequestManagementSection
+                {...defaultTestProps}
+                policy={{
+                  loanable: true,
+                  loansPolicy,
+                  requestManagement,
+                }}
+              />
+            );
+          });
+
+          if (
+            loansPolicy?.profileId === loanProfileMap.ROLLING &&
+            requestManagement?.holds?.renewItemsWithRequest
+          ) {
+            it('should render period', () => {
+              expect(screen.getByTestId('alternateRenewalLoanPeriod')).toBeVisible();
+              expect(Period).toHaveBeenNthCalledWith(
+                periodCallOrderByPlace.alternateRenewalLoan,
+                {
+                  fieldLabel: labelIds.alternateRenewalLoanPeriod,
+                  inputValuePath: 'requestManagement.holds.alternateRenewalLoanPeriod.duration',
+                  selectValuePath: 'requestManagement.holds.alternateRenewalLoanPeriod.intervalId',
+                  intervalPeriods: intervarPeriodsExpectedValue,
+                  changeFormValue: changeMock,
+                }, {}
+              );
+            });
+          } else {
+            it('should not render period', () => {
+              expect(screen.queryByTestId('alternateRenewalLoanPeriod')).not.toBeInTheDocument();
+            });
+          }
+        });
+      });
+    };
+
+    [
+      {
+        describeTitle: 'when "profileId" property value is undefined',
+        loansPolicy: {},
+      },
+      {
+        describeTitle: `when "profileId" property value is "${loanProfileMap.FIXED}"`,
+        loansPolicy: {
+          profileId: loanProfileMap.FIXED,
+        },
+      },
+      {
+        describeTitle: `when "profileId" property value is "${loanProfileMap.ROLLING}"`,
+        loansPolicy: {
+          profileId: loanProfileMap.ROLLING,
+        },
+      },
+    ].forEach(({ describeTitle, loansPolicy }) => {
+      describe(describeTitle, () => {
+        testRequestManagement(loansPolicy);
+      });
+    });
+  });
+
+  describe('"allow recalls to extend overdue loans" field', () => {
+    it('should render field', () => {
+      render(
+        <RequestManagementSection
+          {...defaultTestProps}
+          policy={{
+            loanable: true,
+          }}
+        />
+      );
+
+      expect(Field).toHaveBeenNthCalledWith(
+        fieldCallOrderByPlace.allowRecallsToExtendOverdueLoans,
+        {
+          label: labelIds.allowRecallsToExtendOverdueLoans,
+          name: 'requestManagement.recalls.allowRecallsToExtendOverdueLoans',
+          id: 'requestManagement.recalls.allowRecallsToExtendOverdueLoans',
+          component: Checkbox,
+          type: 'checkbox',
+        }, {}
+      );
+    });
+  });
+
+  describe('"renew items with request" field', () => {
+    it('should render field', () => {
+      render(
+        <RequestManagementSection
```

Comment:
if i dont mistake, you render `RequestManagementSection` with this set of props at least five times. so can we put the cases with same render in one additional `describe` and use `beforeEach` construction? what do you think about it?

Reply:
I already have such describe `when "policy.loanable" is true` but to be honest I would not like to move these tests there. The idea is to test all components separately. We can discuss it.

Reply:
done